src/dashboard/index.js

```diff
@@ -37,6 +38,230 @@ function render() {
         }
       });
   });
+
+    document.getElementById("activity").addEventListener("click", () => {
+        const repository = path.join("file://", __dirname, "activity.html");
+        let os = require("os");
+        fetch(repository)
+            .then((response) => response.text())
+            .then((text) => {
+                document.querySelector("#content").innerHTML = text;
+                var cookbook_activity = document.createElement('div');
+                cookbook_activity.setAttribute("id", "cookbook-activity");
+
+                var container_cookbook_details = document.createElement('div');
```

Comment:
we could use let here

Reply:
Sure we can, they have an almost similar property, and let is es6 property (as it hardly made any difference I used var)